tests/test_tasking.py

```diff
@@ -110,16 +114,56 @@ def test_should_construct_order_parameters(
         } | ({"tags": tags} if tags else {})
         assert order_parameters == expected
 
-
-def test_get_quotations(tasking_mock):
-    get_quotations = tasking_mock.get_quotations()
-    assert len(get_quotations) == 23
-    get_workspace_quotations = tasking_mock.get_quotations(workspace_id=constants.WORKSPACE_ID)
-    assert all(quotation["workspaceId"] == constants.WORKSPACE_ID for quotation in get_workspace_quotations)
-    get_accepted_quotations = tasking_mock.get_quotations(decision=["ACCEPTED"])
-    assert all(quotation["decision"] == "ACCEPTED" for quotation in get_accepted_quotations)
-    get_decided_quotations = tasking_mock.get_quotations(decision=["ACCEPTED", "REJECTED"])
-    assert all(quotation["decision"] in ["ACCEPTED", "REJECTED"] for quotation in get_decided_quotations)
+    @pytest.mark.parametrize("quotation_id", [None, QUOTATION_ID])
+    @pytest.mark.parametrize("workspace_id", [None, constants.WORKSPACE_ID])
+    @pytest.mark.parametrize("order_id", [None, ORDER_ID])
+    @pytest.mark.parametrize("decision", [None, ["NOT_DECIDED", "ACCEPTED", "REJECTED"]])
+    @pytest.mark.parametrize("sort_by", ["updatedAt"])
+    @pytest.mark.parametrize("descending", [False, True])
+    def test_should_get_quotations(
+        self,
+        requests_mock: req_mock.Mocker,
+        tasking_obj: tasking.Tasking,
+        quotation_id: Optional[str],
+        workspace_id: Optional[str],
+        order_id: Optional[str],
+        decision: Optional[List[str]],
+        sort_by: str,
+        descending: bool,
+    ):
+        total_pages = 4
+        query_params: dict[str, Any] = {}
+        sort_order = ",desc" if descending else ",asc"
```

Comment:
let's add this to parameterization

Reply:
I have parametrize descending and set the value according to it. Is that what you meant?